src/transactions/feed/SwapFeedItem.test.tsx

```diff
@@ -124,8 +125,10 @@ describe('SwapFeedItem', () => {
     expect(getElementText(getByTestId('SwapFeedItem/outgoingAmount'))).toEqual('-17.54 cEUR')
   })
 
-  it('still shows balances when feature gate false, hide balances root state true', async () => {
-    jest.mocked(getFeatureGate).mockReturnValue(false)
+  it('still shows balances when feature gate true, hide balances root state true', async () => {
+    jest
+      .mocked(getFeatureGate)
+      .mockImplementation((featureGate) => featureGate === StatsigFeatureGates.USE_TAB_NAVIGATOR)
```

Comment:
nit: are there multiple feature gates accessed in the component under test? If not could we just do mockReturnValue?

Reply:
I know we could, but I saw this in another test and actually like it because its more explicit. just mocking all getFeatureGate calls could lead to some inaccurate tests.

Reply:
I think its not too different, especially the `false` case because it sets the gate in question to `false` and every other gate to true. This pattern is just inverting what other gates return.